### PR TITLE
Another Edit on GitHub link

### PR DESCRIPTION
**Pull request by rgaiacs**
This one replace https://github.com/swcarpentry/lesson-example/issues/99.

@ErinBecker sent a form to the instructors and analysing the response we notice that people want the "Edit on GitHub". This pull request add the link to the navigation bar (on the top) and to the footnote.

# Navigation Bar Screenshot

Before:

nav-old

After:

new-nav

Changes:

- Link to "References" moved to inside "Extras", and
- Added "Edit on GitHub" link.

# Footnote Screenshot

Before:

foot-old

After:

foot-new

Changes:

- Added "Improve this page" link at the begin, and
- Move "Source" link to after "Contributing".

# Proposal timeline

- From 27th February to 3rd March: discussion about this issue. **Please use the reactions if you want to have this merged or not.**
- If positive feedback, between 4th March and 5th March I will move this pull request to the [styles repository](https://github.com/swcarpentry/styles), merge it and send a pull request to the lessons.

### CODE REVIEW

**Comment by vuw-ecs-kevin** [comment]
- Link to "References" moved to inside "Extras", and
- Added "Improve this page" link a
- Added "Edit on GitHub" link.

As someone who has spent a bit of time looking at making the lessons
be renderable "off-line", 

https://github.com/vuw-ecs-kevin/lesson-example/tree/fully-offline-capable

and so spent some time studying the look and feel of the lessons, I think that 
constant additions to items immediatey visble on EVERY page is likely to get 
confusing (or perhaps: add to existing confusion?).

I was not the biggest fan of the menu, yet was told that this was the preferred
way to have ALL of the **auxilliary content** available from ANY page, however,
here is a request is to add more auxilliary content directly to a page, rather than
adding it below an auxilliary menu.

Futherrmore, there is already a Contributing link, and page on the end of
that link, that is visible on EVERY page, and explains ALL of the ways in 
which feedback about lessons can be made, including the use of various 
mechanisms at GitHub.

- Added "Improve this page" link a

The wording of the "Improve this page" item in the footer banner,
doesn't really tell the full story. Clearly the intention is to suggest that
people taking the lesson can "try to help to improve the lesson", although
there's no guarantee that what they would see as an improvement will be 
seen to be one, by those steering the various Carpentry efforts.

I'd also suggest that, within the current Carpentry lesson model, any "Improvement"
would not actually be made to the page from the lesson that was being taken, or 
viewed, because, once a lesson, given at a partilcular time and place has been 
delivered, then surely that lesson instance must be frozen, errata aside, although 
other "forks" of  the content and presentation, then rendered for all future lessons, 
might be expected to be different.

- Link to "References" moved to inside "Extras", and

I feel that the Reference menu item being, effectively, a single item meu title, 
and so making visible, with one click, a page which provides access, from any 
page within the lesson, to a Glossary of potetnailly unfamiliar terms that takers 
of a lesson might be encountering for the first time during the lesson, to be a 
good and useful thing.

To require takers of a lesson to have to click twice to get to such information
seems, to me, sub-optimal.

My preference would be to see both "Edit on GitHub" and "Improve this page" 
links, as suggested, remain within the Contributing page, and where the Contributing 
page is also linked to within the Extras menu, and for References to remain visible on
every page, by being left amongst the main menu titles.

I could also make a case for this by saying that the fewer opportunites (links) given
to move away from the lesson's core content there are, the less likely takers of the
lesson are to "click through" to content that won't help them take on board what is
supposedly being shown/explained to them during the lesson. I am equally sure that
case could be refuted.


- Move "Source" link to after "Contributing".

No preference either way, 

i would suggest though that a link to the "Source" by virtue of targetting something 
at GitHub, could be rendered a duplication by the proposed links that also point to 
GitHub.

Hoping that's useful.

**Comment by lexnederbragt** [comment]
Most people will look at the navigation bar rather than the footer. I suggest to use 'Improve this page' in the navigation bar, and 'edit on github' in the footer.

What is clicking either of those going to lead to? A set of instructions?

**Comment by rgaiacs** [comment]
I discovery that my pull request will not work for lessons write in RMarkdown. Help to fix this is welcomed.

**Comment by rgaiacs** [comment]
> What is clicking either of those going to lead to?

Something like https://github.com/swcarpentry/lesson-example/edit/gh-pages/_episodes/01-design.md but with the correct path.

>  A set of instructions?

No. Instructions are on the "Contributing" link at the footnote.

File: _includes/lesson_footer.html [hunk 1]
```diff
@@ -17,10 +17,12 @@
     </div>
     <div class="col-md-6" align="right">
       <h4>
-	<a href="{{ site.github.repository_url }}/">Source</a>
+        <a href="{{site.github.repository_url}}/blob/gh-pages/{{page.path}}">Improve this page <span class="glyphicon glyphicon-pencil" aria-hidden="true"></span></a>
```
Comment (rgaiacs) [inline _includes/lesson_footer.html]: Change `/blob/` to `/edit/`.

**Pull request closed without merging**